jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/StatisticsHandler.java

```diff
@@ -422,19 +437,6 @@ public Content.Chunk read()
             {
                 return _errorContent != null ? _errorContent : super.read();
             }
-
-            @Override
-            public WrapperProcessor wrapProcessor(Processor processor)
-            {
-                _statisticsRequest = (StatisticsRequest)processor;
-                return super.wrapProcessor(processor);
-            }
-
-            @Override
-            public void process(Request ignored, Response response, Callback callback) throws Exception
-            {
-                super.process(ignored, new MinimumDataRateResponse(this, response), callback);
-            }
```

Comment:
Another good example of confusing code this PR resolves.   The current usage style often results in two copies of the request in scope, with one needing to be ignored.   This is confusing and in this case has resulted in a wrong name for a request that is obviously not ignored.